thirdparty/github.go

```diff
@@ -12,9 +12,11 @@ import (
 	"time"
 
 	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/cache"
```

Comment:
Random comment placement: 
I had two general questions: 
How will the ttl index be set up, aka after what amount of time will the cache expire? We might want to tweak it until we get it right, so it might be a good idea to document somewhere what it's set to and how to change it.

Once we move over to github App, will we keep this caching or remove it?

Reply:
😆 .
I was thinking like a week. Like probably most things won't be useful after that. I mean, it's not really a big deal if we keep things too long unless the storage becomes an issue. And if we kill things too fast the worst case is that we'll ask GH for it again.

Reply:
It seems like there's not much of a downside to using the cache and if it saves requests then why remove it? 
I think the decision will hinge on whether we see it effectively reducing the number of calls that count against the rate limit.

Reply:
Are we not worried about the data becoming outdated?

Reply:
That makes sense, thank you!